### PR TITLE
Created demo

### PR DESCRIPTION
**Pull request by pgodfrey**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,94 @@
 # kaazing.io-demo-stocks
 Stocks demo for kaazing.io
+
+This project was bootstrapped with [Create React App](https://github.com/facebookincubator/create-react-app).
+
+You can find the full version of the Create React App guide [here](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md).
+
+## Available Scripts
+
```
Comment (robinzimmermann) [inline README.md]: Provide instructions for building, e.g. `yarn` and / or `npm install`.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,94 @@
...
+Runs the app in the development mode.<br>
+Open [http://localhost:3000](http://localhost:3000) to view it in the browser.
+
+The page will reload if you make edits.<br>
+You will also see any lint errors in the console.
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode.<br>
+See the section about [running tests](#running-tests) for more information.
```
Comment (robinzimmermann) [inline README.md]: There is no "running tests" section, the link doesn't work.

**Comment by robinzimmermann** [comment]
- Make the column headers for **Symbol** and **Name** left-aligned to match the data.

- Add some padding to big background rectangle so the symbol names on the right and the the change prices on the right don't look so squashed up against the sides.

- Can the downward facing red triangles be moved down a pixel or two so they align with the top of the numbers better? Like the baseline of the green triangles are aligned with the baseline of the green numbers.

- Tab title should not say "React App"

- Change the favicon to the attached image:

favicon-32x32

- Heading on the page should have a lowercase `k` for `kaazing.io`.

- The Javascript console has an error. Is it possible to eliminate that?

- See other comments inline, in the review (in the README file)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,16 @@
+{
+  "name": "my-app",
```
Comment (robinzimmermann) [inline package.json]: Put correct values in `package.json` for `name` and `description`.

File: public/manifest.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
```
Comment (robinzimmermann) [inline public/manifest.json:1]: Files from the `public` directory should not be checked in, right?

File: src/App.css [hunk 5]
```diff
@@ -0,0 +1,96 @@
+html,
+body {
+	height: 100%;
+}
+
+body {
```
Comment (robinzimmermann) [inline src/App.css]: Do we need two `body` tags right next to each other?

File: src/App.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+import './App.css';
+
+class Stocks extends Component {
+  constructor(props) {
+    super(props);
+    
+    this.state = {stocks: new Map()};
+    
+    // Connect to Kaazing.io SSE
+    let eventSource = new EventSource("https://a8uofb8l.streaming.kaazing.io/topic/sandbox.stocks");
```
Comment (robinzimmermann) [inline src/App.js]: The public endpoint `https://streams.kaazing.net/sandbox/stocks` is now available to use here.

**Review by robinzimmermann — CHANGES_REQUESTED** [review]
- Make the column headers for **Symbol** and **Name** left-aligned to match the data.

- Add some padding to big background rectangle so the symbol names on the right and the the change prices on the right don't look so squashed up against the sides.

- Can the downward facing red triangles be moved down a pixel or two so they align with the top of the numbers better? Like the baseline of the green triangles are aligned with the baseline of the green numbers.

- Tab title should not say "React App"

- Change the favicon to the attached image:

favicon-32x32

- Heading on the page should have a lowercase `k` for `kaazing.io`.

- The Javascript console has an error. Is it possible to eliminate that?

- See other comments inline, in the review

- We need to add a license
    - Add an appropriate license that gives permission for people to use without attribution.
    - Add a Kaazing copyright to key source files.

File: public/manifest.json (hunk 4 above)
Comment (pgodfrey) [reply public/manifest.json:1]: I think they do need to be checked in - it contains the index.html, manifest and favicon

**Pull request merged**